src/CADL.Extension/Emitter.Csharp/package.json

```diff
@@ -36,14 +36,14 @@
         "dist/**"
     ],
     "devDependencies": {
+        "@azure-tools/cadl-azure-core": "0.6.0",
```

Comment:
It is better to keep the order as before, just to update the version. In that way, the diff may more clear about what is changed.

Reply:
I'm trying to make it in alphabetical order.